--- r8/src/main/java/com/android/tools/r8/graph/DexItemFactory.java	2018-12-12 18:04:21.067534807 -0500
+++ r8/src/main/java/com/android/tools/r8/graph/DexItemFactory.java	2018-12-12 20:38:31.455902442 -0500
@@ -787,7 +787,7 @@
   public synchronized List<Marker> extractMarkers() {
     // This is slow but it is not needed for any production code yet.
     List<Marker> markers = new ArrayList<>();
-    for (DexString dexString : strings.keySet()) {
+    for (DexString dexString : ((Map) strings).keySet()) {
       Marker marker = Marker.parse(dexString);
       if (marker != null) {
         markers.add(marker);
